### PR TITLE
Bug 1319610 - Aggregate process and gpu scalars

### PR DESCRIPTION
**Pull request by fbertsch**
@chutten r?

### CODE REVIEW

File: tests/test_aggregator.py [hunk 1]
```diff
@@ -104,14 +104,14 @@ def test_numerical_scalars():
                     metric = "{}_{}".format(metric, label)
 
                 metric_count[metric][process_type] += 1
-                assert(value["count"] == expected_count(process_type))
+                assert value["count"] == expected_count(process_type, True), "Expected {}, Got {}, Process {}".format(expected_count(process_type, True), value["count"], process_type)
```
Comment (chutten) [inline tests/test_aggregator.py]: looks like we're missing an open-paren?

File: mozaggregator/aggregator.py [hunk 2]
```diff
@@ -228,7 +228,14 @@ def _extract_simple_measures(state, simple, process_type="parent"):
         elif isinstance(value, (int, float, long)):
             _extract_scalar_value(state, u"_".join((SIMPLE_MEASURES_PREFIX, name.upper())), u"", value, SIMPLE_MEASURES_LABELS, process_type)
 
-def _extract_numeric_scalars(state, scalar_dict):
+
+def _extract_scalars(state, process_payloads):
+    for process in ("parent", "content", "gpu"):
```
Comment (chutten) [inline mozaggregator/aggregator.py]: I guess we should probably pull this someplace global, right? (even though it's only used the once)

File: tests/test_aggregator.py (hunk 1 above)
Comment (fbertsch) [inline tests/test_aggregator.py]: When you start adding print strings for asserts, the parens become less useful. Basically here I'd have to put: `assert(my_assertion), "print str"` which arguably looks worse.

Comment (chutten) [inline tests/test_aggregator.py]: Fair enough.

**Comment by fbertsch** [comment]
@chutten string scalars are ignored, and boolean scalars TBD. @Dexterp37, when reading a boolean scalar in Python with the Dataset API, will the type be (0, 1) or (False, True)?

**Comment by Dexterp37** [comment]
@fbertsch , boolean scalars are serialized as booleans in the ping JSON payload, so they should be converted to a Boolean (False, True) unless |get_ping_properties| does something weird. You should probably double check that.

**Comment by fbertsch** [comment]
Per a discussion with @Dexterp37 this *should* work as-is for Boolean, and we can revisit once one is implemented.

**Pull request merged**